studio/lib/payloads/compensations.ts

```diff
@@ -31,5 +38,6 @@ export interface CompensationsPage {
   page: string;
   slug: Slug;
   benefitsByLocation: BenefitsByLocation[];
+  salaries: SalariesByLocation[];
```

Comment:
I see we're doing salaries here, but benefitsByLocation above. Let's do either or

Reply:
That requires a breaking schema change, do we want to do it in this PR?

Reply:
No not at all. We can do it in a cleanup?